prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -104,9 +104,12 @@ size_t rxr_pkt_req_data_offset(struct rxr_pkt_entry *pkt_entry)
 {
 	int pkt_type, read_iov_count;
 	size_t pkt_data_offset;
+	struct rxr_base_hdr *base_hdr;
 
-	pkt_type = rxr_get_base_hdr(pkt_entry->pkt)->type;
-	pkt_data_offset = rxr_pkt_req_hdr_size(pkt_entry);
+	base_hdr = rxr_get_base_hdr(pkt_entry->pkt);
+	pkt_type = base_hdr->type;
+	pkt_data_offset = rxr_pkt_req_header_size(pkt_type, base_hdr->flags,
```

Comment:
Can we use the name `rxr_pkt_req_hdr_size`?  Because we use `hdr` in almost all other places.

Reply:
Sure, will do.

Reply:
Done. Also renamed `rxr_pkt_max_header_size` and `rxr_pkt_max_req_header_size`